### DISCUSSION TITLE
ppi_client: Cuentas con saldo en Sandbox

### DISCUSSION THREAD

**Discussion by rodrigomocca** [itAtPPI/ppi-connector-python#9] [General]
Buenas,
Hay alguna forma de que las cuentas de Sandbox tengn saldo superior a 0?
Me gustaria probar hacer una orden, pero solo lo puedo probar en produccion, dado que en sandbox no tengo dinero disponible.

Gracias.

**Comment by Seba-Beron** [comment]
Hola, encontraste la forma? estoy viendo de hacer lo mismo
